python/private/config_settings.bzl

```diff
@@ -178,40 +39,70 @@ def construct_config_settings(name = None):  # buildifier: disable=function-docs
         # also prevents version-unaware targets from inadvertently matching
         # a select condition when they shouldn't.
         build_setting_default = "",
-        values = [""] + VERSION_FLAG_VALUES.keys(),
+        visibility = ["//visibility:public"],
+    )
+
+    _python_major_minor(
+        name = "python_version_major_minor",
```

Comment:
name is up for bike shedding.

Reply:
I think the name is fine -- just make it more internal sounding -- prefix it with "_". It's not a flag, and shouldn't be directly set, so lets just hide it as best we can so it isn't an attractive nuisance. 

(or move it under private somewhere, but we have other internal names here, so i think an underscore is sufficient).